.github/workflows/ci.yml

```diff
@@ -55,6 +55,8 @@ jobs:
       - name: Install dependencies
         run: |
           pip install ".[dev]"
+          solc-select use 0.4.25 --always-install
+          solc-select use 0.8.0 --always-install
```

Comment:
Looks good, almost there. You'll have to do something similar to this to get the IR tests passing eg update it to look something like this:

```
      - name: Install dependencies
        run: |
          pip install ".[dev]"
          solc-select install 0.5.0
          solc-select use 0.8.11 --always-install

      - name: Install old solc
        if: matrix.os == 'ubuntu-latest'
        run: solc-select install 0.4.0
```
IR tests require 0.5.0 and 0.4.0 except not on windows because that binary is not available.
Note: only one solc version can be `use`d at a time, so you can also just `install` multiple solc versions and only `use` the last one

Reply:
ok got it thankyou